test/e2e_node/hugepages_test.go

```diff
@@ -85,7 +85,14 @@ func makePodToVerifyHugePages(baseName string, hugePagesLimit resource.Quantity)
 
 // configureHugePages attempts to allocate 10Mi of 2Mi hugepages for testing purposes
 func configureHugePages() error {
-	err := exec.Command("/bin/sh", "-c", "echo 5 > /proc/sys/vm/nr_hugepages").Run()
+	// Drop all kernel caches to make sure we can get the requested amount of huge pages
+	// This will only affect kernel level cache hit, and nothing visible in userspace
+	// https://www.kernel.org/doc/Documentation/sysctl/vm.txt
+	err := exec.Command("/bin/sh", "-c", "echo 3 > /proc/sys/vm/drop_caches").Run()
```

Comment:
Are the k8s tests running as `root`? I'm a little surprised that a test would have permissions to run this command (especially on the k8s CI)?

Reply:
Hi,
Yes, and they have to!

The same permissions apply to the pre allocation of huge pages (see 2-3 lines below) by writing to `/proc/sys/vm/nr_hugepages` :smile: